### PR TITLE
Integrating examples from the "Demystifying Microservices" guide.

### PR DESCRIPTION
**Pull request by dheffelfinger**

### CODE REVIEW

File: payara-micro/demystifying-microservices-example/CrudDatabase/nbactions.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MattGill98) [inline payara-micro/demystifying-microservices-example/CrudDatabase/nbactions.xml]: Has this been accidentally added?

File: payara-micro/demystifying-microservices-example/CrudView/src/main/webapp/js/libs/jquery/jquery.js [hunk 2]
```diff
@@ -0,0 +1,10220 @@
+/*!
```
Comment (MattGill98) [inline payara-micro/demystifying-microservices-example/CrudView/src/main/webapp/js/libs/jquery/jquery.js]: Is it possible to remove the jQuery and Bootstrap libraries in favour of using a CDN?

File: payara-micro/demystifying-microservices-example/CrudView/pom.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>fish.payara</groupId>
+    <artifactId>CrudView</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+    <name>CrudView</name>
+    <parent>
```
Comment (MattGill98) [inline payara-micro/demystifying-microservices-example/CrudView/pom.xml]: Just for convention's sake would you mind putting the parent at the top?

Comment (dheffelfinger) [reply payara-micro/demystifying-microservices-example/CrudView/pom.xml]: Checked a few other pom files in the repo, all of them have the <parent> tag under name, seems to be consistent with other pom files. Leaving as-is for now.

See:
microprofile/pom.xml
microprofile/rest-client/pom.xml
payara-micro/jpa-datasource-example/pom.xml

**Comment by MattGill98** [comment]
**NOTE TO MERGER** This will need to be squash merged to remove the dependency JARs from Git history

**Pull request merged**